torch/ao/quantization/pt2e/export_utils.py

```diff
@@ -136,20 +136,29 @@ def bn_eval(
         torch.randn(1),  # bn_running_mean
         torch.randn(1),  # bn_running_var
     )
+
+    # TODO: We shouldn't assume that the model is on cuda just because cuda is
+    # available, but right now there's no other way to differentiate between
+    # the different bn ops. Once batch norm consolidation lands, we won't have
+    # to do this anymore: https://github.com/pytorch/pytorch/pull/119496.
+    is_cuda = torch.cuda.is_available()
```

Comment:
right, can we get this from the model itself?

Reply:
e.g. with this: https://github.com/pytorch/pytorch/blob/main/torch/ao/quantization/fx/utils.py#L225

Reply:
oh this is great, I'll update with this util